doc/neps/scope.rst

```diff
@@ -0,0 +1,41 @@
+==============
+Scope of NumPy
+==============
+
+Here, we describe aspects of N-d array computation that are within scope for NumPy development. This is *not* an aspirational definition of where NumPy should aim, but instead captures the status quo—areas which we have decided to continue supporting, at least for the time being.
+
+- **In-memory, N-dimensional, homogeneously typed (single pointer + strided) arrays on CPUs**
+
+  - Support for a wide range of data types
+  - Not specialized hardware such as GPUs
+  - But, do support wide range of CPUs (e.g. ARM, PowerX)
+
+- **Higher level APIs for N-dimensional arrays**
+
+  - NumPy is a *de facto* standard for array APIs in Python
+  - Indexing and fast iteration over elements (ufunc)
+  - Interoperability protocols with other data container implementations (like `__array_ufunc__`).
+
+- **Python API and a C API** to the ndarray's methods and attributes.
+
+- Other **specialized types of N-dimensional arrays**:
+
+  - Masked arrays
```

Comment:
Add `np.memmap`?

Reply:
I guess that makes sense. I never thought of it as a different type of array, more like a memory optimization trick for ndarrays with some interesting gotchas.